app/views/articles/edit.html.erb

```diff
@@ -6,6 +6,6 @@
   <%= form.label :content %>
   <%= form.text_area :body, id: 'article_content', value: @article.body %>
   <%= form.label :categories %>
-  <%= form.select(:categories, Category.all.collect {|c| [ c.name, c.id ] }, {}, { multiple: true, id: 'categories' })%>
+  <%= form.select(:categories, Category.all.collect {|c| [ c.name, c.id ] }, {}, { multiple: true, id: 'Categories' })%>
```

Comment:
We don't set id's with capitals

Reply:
If the test is not passing, use `downcase` in your step definition

Reply:
okay